src/main/resources/liquibase/crop_changelog/21_5_0.xml

```diff
@@ -34,4 +34,135 @@
 		</sql>
 	</changeSet>
 
+	<changeSet author="cuenyad" id="v21.5.0-3">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">select count(*) from cvterm where name = 'STABrAPP BLUES - LME4' and cv_id = 1020;</sqlCheck>
```

Comment:
@cuenyad use is_obsolete as part of the condition (check by unique keys) in all cases

Reply:
Done.